common/common.h

```diff
@@ -192,4 +192,6 @@ typedef long long ssize_t;
 typedef unsigned short mode_t;
 #endif
 
+#define UNUSED(x) (void)(x)
```

Comment:
I'd prefer `#define MP_UNUSED(...) (void)(__VA_ARGS__)` so we can put multiple on the same line.